packages/client/bin/utils.ts

```diff
@@ -551,7 +551,8 @@ async function inputAccounts(args: ClientOpts) {
   })
 
   // Hide key input
-  ;(rl as any).input.on('keypress', function () {
+  // @ts-expect-error -- Absent from type
+  rl['input'].on('keypress', function () {
```

Comment:
Why is this absent? :thinking: I know it was so originally but is this `rl` object not (correctly) typed?